ci/qa/phpcpd

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+#cd $(dirname $0)/../../
+#
+## https://github.com/sebastianbergmann/phpcpd
+#./vendor/bin/phpcpd --exclude src/*/Tests/* src
+
+# PHPCS is currently disabled due to Xdebug throwing a fatal error.
+echo "Skipping PHPCPD due to Xdebug fatal error"
```

Comment:
There are no JS dependencies in this repository. so it would mean that a bunch of yarn boilerplate code would be added for just one library. Scrutinize should do the same as PHPCPD.

Reply:
I can agree with your reasoning. However we have Yarn on the test container. But I agree, having a CI only test is not very predictable. And having one single dev-only JS dependency is not that cute either..